lms/djangoapps/grades/management/commands/recalculate_learner_grades.py

```diff
@@ -0,0 +1,54 @@
+"""
+Command to recalculate a user's grades for a course, for every user in
+a csv of (user, course) pairs.
+"""
+
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import csv
+import logging
+
+from django.contrib.auth.models import User
+from django.core.management.base import BaseCommand
+
+from lms.djangoapps.grades.signals.handlers import recalculate_course_and_subsection_grades
+
+log = logging.getLogger(__name__)
+
+DATE_FORMAT = "%Y-%m-%d %H:%M"
+
+
+class Command(BaseCommand):
+    """
+    Example usage:
+        $ ./manage.py lms recalculate_learner_course_grades --csv learner_courses_to_recalculate.csv
+    """
+    help = 'Recalculates a user\'s grades for a course, for every user in a csv of (user, course) pairs'
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--csv',
+            dest='csv',
+            help='A CSV containing (user, course) pairs.',
+            required=True
+        )
+
+    def handle(self, *args, **options):
+        filename = options['csv']
+
+        with open(filename) as csv_file:
+            self.parse_csv_file(csv_file)
+
+    def parse_csv_file(self, csv_file):
+        csv_reader = csv.DictReader(csv_file)
+        csv_reader.next()
+
+        for row in csv_reader:
+            try:
+                user = User.objects.get(
+                    id=row['user_id']
+                )
+                course_key = row['course_id']
+                recalculate_course_and_subsection_grades(None, user, course_key)
```

Comment:
You can replace this with a call to the task itself (then you won't need to load the user, either):
```python
from lms.djangoapps.grades.tasks import recalculate_course_and_subsection_grades_for_user
recalculate_course_and_subsection_grades_for_user.apply_async(
    kwargs={'user_id': row['user_id'], 'course_key': row['course_id']}
)
```